### DISCUSSION TITLE
Memory leak when frequently call EventsEmit

### DISCUSSION THREAD

**Discussion by nealwon** [wailsapp/wails#3553] [Q&A]
I create a empty project with an official template, I only changed a bit code below:
The Go code:
```go
for {
       rt.EventsEmit(a.ctx, "gogogo", "hell world")
        time.Sleep(time.Microsecond * 1)
}
```
The JS code:
```javascript
window.runtime.EventsOn('gogogo', function(data) {
  console.log(data)
});
```
When the Go code start running, the memory always increases. 
It start at about 30MB,and 10min later,the memory is 80MB.and 30 min later ,the memory is 120MB.

My question is:
Why the memory is not freed?

My Environment Information:

# Wails
Version | v2.9.1

# Dependencies

| Dependency                | Package Name | Status    | Version         |
| ---- | ---- | ---- | ---- |
| Xcode command line tools  | N/A          | Installed | 2395            |
| Nodejs                    | N/A          | Installed | 20.12.2         |
| npm                       | N/A          | Installed | 10.5.0          |
| *Xcode                    | N/A          | Installed | 13.2.1 (13C100) |
| *upx                      | N/A          | Installed | upx 4.1.0       |
| *nsis                     | N/A          | Installed | v3.10           |

**Comment by tianya66** [comment]
有人看一下这个问题吗
